pom.xml

```diff
@@ -330,6 +330,7 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.10</version>
         <configuration>
+          <redirectTestOutputToFile>true</redirectTestOutputToFile>
```

Comment:
This was an accidental addition. I like to be able to go directly to the output of a test without scrolling so much locally, but it doesn't work with Travis. I'll remove it.